kots-exporter/kots-exporter.sh

```diff
@@ -346,8 +343,32 @@ rm_kots_annot_label_resources(){
     echo "Kots removal logs are available - $path/logs/$kotsCleanupLogFile"
 
     echo "Deleting job inject-bottoken-xxxx"
-    kubectl -n $namespace delete "$(kubectl -n $namespace get jobs -o name | grep inject)" \
-    || echo "Manually run the command: kubectl -n $namespace delete $(kubectl -n $namespace get jobs -o name | grep inject)"
+    if [ "$(kubectl -n $namespace get jobs -o name | grep inject)" != "" ]; then
+    kubectl -n $namespace delete "$(kubectl -n $namespace get jobs -o name | grep inject)"
+    fi
+}
+
+postgres_migration(){
+    if "$POSTGRES_INTERNAL"; then
+    echo ""
+    echo "############ PREPARING POSTGRES FOR MIGRATION ############"
+
+    echo "Upgrading to server CircleCI Server 4.0 includes upgrading the Postgres chart"
```

Comment:
I think server needs to be lowercase here

```suggestion
    echo "Upgrading to server CircleCI server 4.0 includes upgrading the Postgres chart"
```